docs/index.rst

```diff
@@ -44,6 +44,23 @@ respectively.
 Finally, this package provides :func:`~casa_formats_io.coordsys_to_astropy_wcs`) which can
 be used to convert CASA WCS information to :class:`~astropy.wcs.WCS` objects.
 
+Table reader (experimental)
+---------------------------
+
+This package includes an experimental generic table reader which integrates with
+the astropy :class:`~astropy.table.Table` class. To use it, first import
+the ``casa_formats_io`` module, which registers the reader, then use the
+:meth:`Table.read <astropy.table.Table.read>` method::
+
+    >>> import casa_formats_io
+    >>> from astropy.table import Table
+    >>> table = Table.read('my_dataset.ms', format='casa-table')
```

Comment:
could one do:
```python
from casa_formats_io import Table
table = Table.read('my_dataset.ms', format='casa-table')
```
and still have `Table` be the astropy default table?  I think this small bit of convenience wrapping would be nice.  I wouldn't suggest changing the docs for this, but perhaps add it.